stream/src/main/scala/org/apache/pekko/stream/impl/fusing/Ops.scala

```diff
@@ -267,9 +266,12 @@ private[stream] object Collect {
         } catch {
           case NonFatal(ex) =>
             decider(ex) match {
-              case Supervision.Stop    => failStage(ex)
-              case Supervision.Resume  => if (!hasBeenPulled(in)) pull(in)
-              case Supervision.Restart => if (!hasBeenPulled(in)) pull(in)
+              case Supervision.Stop => failStage(ex)
+              case _                =>
+                // The !hasBeenPulled(in) check is not required here since it
```

Comment:
@He-Pin Is this comment correct?

Reply:
Is true, but where this comments come from?

Reply:
As a maintainer of Pekko, I have permission to add commits onto branches of other people's PR. This is my commit that I added now.

I will approve PR and squash + merge it, thanks for work!

Reply:
I see, hah, The chinese networking to github is very unstable, that's why I hate changing and changing, sometime I can't pulling pull too, both via ssh and https